src/xrGame/gsc_dsigned_ltx.cpp

```diff
@@ -76,9 +76,7 @@ bool gsc_dsigned_ltx_reader::load_and_verify(u8* buffer, u32 const size)
     xr_strcat(dsign_section, dsign_sect_size, ltx_date.c_str());
     u32 new_size = size - dsign_sect_size + ltx_date.size() + 1; // xr_strlen(buffer)
     if (!verify(buffer, new_size, ltx_dsign))
-    {
         return false;
-    }
```

Comment:
It's better to insert one empty line here, after `return false;` (but not before `if (!verify(...))`)